tests/foreman/ui/test_errata.py

```diff
@@ -156,13 +156,13 @@ def test_positive_list(self):
         with Session(self.browser) as session:
             session.nav.go_to_errata()
             self.errata.show_only_applicable(False)
-            self.assertIsNone(self.errata.search(CUSTOM_REPO_ERRATA_ID))
-            self.assertIsNotNone(self.errata.search(FAKE_1_ERRATA_ID))
+            self.assertIsNone(self.errata.search(FAKE_3_ERRATA_ID))
+            self.assertIsNotNone(self.errata.search(CUSTOM_REPO_ERRATA_ID))
```

Comment:
This change is basically bonus. When we changed from sat6 tools repo to sat6.2 tools repo errata ids became outdated so the test was failing as it couldn't find mentioned errata ids.